code/GrubsGame.cs

```diff
@@ -101,6 +101,23 @@ public override void OnVoicePlayed( IClient client )
 		UI.PlayerList.Current?.OnVoicePlayed( client );
 	}
 
+#if DEBUG

```

Comment:
DEBUG pre-processor symbols currently don't get removed while uploading to asset.party. Can we ensure this is only available for development through another means?

Reply:
https://github.com/sboxgame/issues/issues/3104

Reply:
Ah yeah, I assumed that issue was fixed by now. Oops. I'll figure something else out instead.